CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 - **json:** Serialize jsonpath return ([#5921](https://github.com/ScoopInstaller/Scoop/issues/5921))
 - **scoop-search:** Catch error of parsing invalid manifest ([#5930](https://github.com/ScoopInstaller/Scoop/issues/5930))
+- **autoupdate:** Copy `PSCustomObject`-type properties within `autoupdate` to prevent reference changes. ([#5934](https://github.com/ScoopInstaller/Scoop/issues/5934))
```

Comment:
```suggestion
- **autoupdate:** Copy `PSCustomObject`-type properties within `autoupdate` to prevent reference changes ([#5934](https://github.com/ScoopInstaller/Scoop/issues/5934))
```